spring-cloud-gcp-starters/spring-cloud-gcp-starter-storage/pom.xml

```diff
@@ -26,5 +26,10 @@
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-gcp-storage</artifactId>
 		</dependency>
```

Comment:
Well, I don't if that is the point, but I decided to restore the whole scenario from the issue. But, yeah, sometimes we may not meet issue until we make the test web-based.

The problem here is fundamental: having the `ProtocolResolver` as a bean is too late for resource resolving.